.github/stale.yml

```diff
@@ -0,0 +1,83 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+pulls:
+  # XXX: We'll only match very few issues to ensure the bot works as expected
+  # XXX: Once working, we're going to reduce the stale window incrementally
+  daysUntilStale: 1250
+
+  # Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
```

Comment:
I was confused by that "Issue or Pull Request" since this part seems to be purely about PRs.

Reply:
And the next section, for issues, also has comments talking about both issues and PR.

Might be worth cleaning that up to clear all confusion.

Reply:
Ha, never thought about that. These were the comments of the configuration template. But yeah, this is indeed confusing if you don't know that part. Also fixed one occurrence in the "to be posted comments".